source/infra/review-rdo-infra-core.html.md

```diff
@@ -0,0 +1,46 @@
+---
+title: review.rdoproject.org Infrastructure core administrators
+category: documentation,infrastructure
+---
+
+# review.rdoproject.org infrastructure core administrators
+
+The `config-core` group for review.rdoproject.org takes care of managing all configuration for the Software Factory instance used by the RDO project.
+
+## Current core list
+
+* Alan Pevec
+* Attila Darazs
+* David Moreau Simard
+* Fabien Boucher
+* Frederic Lepied
+* Javier Peña
+* Tristan Cacqueray
```

Comment:
Maybe worth to mention rdo-provenpackagers group here (for release-related changes. E.g creating packages repo, CBS jobs etc.). Well, you explain later that people are expected to approve changes within their expertise field so maybe not worth to give much details.

Reply:
I've added a note about it, let me know what you think of the next version.